xenia-build

```diff
@@ -1690,6 +1697,9 @@ class DevenvCommand(Command):
                 print('ERROR: Visual Studio is not installed.');
                 return 1
             print('Launching Visual Studio...')
+        elif sys.platform == 'darwin':
+            print('Launching Xcpde...')
```

Comment:
This part is a duplicate of #1895 (there is a comment about the typo in the discussion of that pull request) — they should be separate, or this pull request should be superseding the xenia-build one.